### PR TITLE
docs: Add readme for Power Monitoring Operator

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR adds the readme for Power Monitoring Operator

### CODE REVIEW

File: ci-operator/config/openshift-power-monitoring/power-monitoring-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+### Test Setup, Execution and Reporting Results
+
+Following the test cluster provisioning, the below step are executed:
+
+[`openshift-power-monitoring-tests-kepler`](../../../step-registry/openshift-power-monitoring/tests/kepler/README.md)
+
+## Custom Images
+
+- `kepler-operator`
```
Comment (vimalk78) [inline ci-operator/config/openshift-power-monitoring/power-monitoring-operator/README.md:49]: change to `power-monitoring-operator`

File: ci-operator/step-registry/openshift-power-monitoring/tests/kepler/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Purpose
+
+To execute the `Power Monitoring Operator` [E2E tests](<https://github.com/openshift-power-monitoring/power-monitoring-operator/tree/v1alpha1/tests>) using the provided arguments. All the results will be copied into Artifact directory.
+
+## Process
+
+- Create a directory `/$ARTIFACT_DIR/test-run-logs` to store the test results.
+- Validate the operator installation by checking the `rollout status` and waiting for `controller` deployment to be available.
+- Execute the Operator e2e tests and capture the run inside `test-run-logs` directory. Log the events for `openshift-operators` as well as `openshift-kepler-operator` namespace while test run is executing.
```
Comment (vimalk78) [inline ci-operator/step-registry/openshift-power-monitoring/tests/kepler/README.md:19]: in the near future, the name of the namespace will also change to `openshift-power-monitoring`

Comment (vprashar2929) [reply ci-operator/step-registry/openshift-power-monitoring/tests/kepler/README.md:19]: Yes will update this once we update tests as well.

**Pull request merged**